modules/core/src/lib/attribute-transition-manager.js

```diff
@@ -241,7 +251,7 @@ export default class AttributeTransitionManager {
         data: new Float32Array(toLength)
       });
     }
-    padBuffer({fromState, toState, fromLength, toLength, context: this.context});
+    padBuffer({fromState, toState, fromLength, toLength, opts});
```

Comment:
The `opts` argument here still seems like a `context`, and an odd API design overall. Is it even used inside `padBuffer`?